dynamics-nav/Object-Specifications-and-Limitations.md

```diff
@@ -1,5 +1,6 @@
 ---
 title: "Object Specifications and Limitations"
+description: Discusses object specifications and l limitations for the database, tables, pages and reports, and Codeunits.
```

Comment:
Discusses object specifications and limitations for the database, tables, pages, reports, and Codeunits.